pinot-segment-local/src/main/java/org/apache/pinot/segment/local/startree/v2/builder/MultipleTreesBuilder.java

```diff
@@ -165,6 +231,13 @@ private static SingleTreeBuilder getSingleTreeBuilder(StarTreeV2BuilderConfig bu
 
   @Override
   public void close() {
+    if (_separatorTempDir != null) {
+      try {
+        FileUtils.forceDelete(_separatorTempDir);
+      } catch (Exception e) {
+        LOGGER.warn(e.getMessage(), e);
```

Comment:
(minor) Put a more meaningful message, e.g. `Caught exception when deleting the separator tmp directory: {}`